packages/lisk-p2p/src/p2p.ts

```diff
@@ -200,13 +202,18 @@ export class P2P extends EventEmitter {
 		this._peerPool = new PeerPool({
 			connectTimeout: this._config.connectTimeout,
 			ackTimeout: this._config.ackTimeout,
+			peerSelectionForSendRequest: config.peerSelectionForSendRequest,
```

Comment:
should we assign default here? i feel like it's more clear than assign the default internally

Reply:
Yes, it's better to assign it in P2P constructor. I have made the relevant changes.